torch/_inductor/package/package.py

```diff
@@ -284,5 +286,6 @@ def load_package(path: Union[str, io.BytesIO], model_name: str = "model") -> AOT
             loader = torch._C._aoti.AOTIModelPackageLoader(f.name, model_name)  # type: ignore[call-arg]
             return AOTICompiledModel(loader)
 
+    path = os.fspath(path)
```

Comment:
for the runtime changes like this it would be nice to have split them out separately

Reply:
Yea, but often changing the annotations requires some code changes to avoid type errors. For this one, it's because `torch._C._aoti.AOTIModelPackageLoader` only supports `str`.